src/flameGraph.cpp

```diff
@@ -26,8 +26,8 @@
 // Browsers refuse to draw on canvas larger than 32767 px
 const int MAX_CANVAS_HEIGHT = 32767;
 
-INCBIN(FLAMEGRAPH_TEMPLATE, "flame.html")
-INCBIN(TREE_TEMPLATE, "tree.html")
+INCBIN(FLAMEGRAPH_TEMPLATE, "src/res/flame.html")
+INCBIN(TREE_TEMPLATE, "src/res/tree.html")
```

Comment:
Did you test it with XCode on macOS?

Is `-I src/res` include path no longer needed?

Reply:
> Did you test it with XCode on macOS?

I tested it with XCode (creating a project that uses the external build script) and it works.

> Is -I src/res include path no longer needed?

You're right, it is no longer needed.